db.json

```diff
@@ -0,0 +1,30 @@
+{
+  "users": [
```

Comment:
It would be useful to set this file on the .gitignore to prevent it being uploaded and overwritten each time.